Vagrantfile

```diff
@@ -27,6 +27,6 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provision "shell", inline: <<-SHELL
-    sudo yum install -y git livecd-tools
+    sudo yum install -y git livecd-tools docker parted
```

Comment:
Is parted related to resolving the SELinux issue? I can understand installing the docker package for the policy files.